generators/Exercises/LargestSeriesProduct.cs

```diff
@@ -0,0 +1,13 @@
+using Generators.Input;
+
+namespace Generators.Exercises
+{
+    public class LargestSeriesProduct : Exercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            foreach (var canonicalDataCase in canonicalData.Cases)
+                canonicalDataCase.Property = "GetLargestProduct";
```

Comment:
Maybe `CalculateLargestSeriesProduct` would be a better name.

Reply:
Hmm. Maybe?

I interpret `GetLargestProduct` as *I wanna get the largest product in this series*, which fits the exercise description. Basically the same as saying `CalculateLargestSeriesProduct`, but more precise. I would leave it, but regardless, both convey the correct meaning.

Reply:
Well argued. I agree